### PR TITLE
Recycle page

### PR DESCRIPTION
**Pull request by IgorOsiyuk**
create new page which contains list of keeps that was removed from main board 
image

create sidebar for change application pages 
image
also moved button logout into sidebar and rewrite it for a separate component

### CODE REVIEW

File: google_keeps/src/components/RecycleComponent/RecycleBlock.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    padding: 16px;
+  }
+`;
+
+const RecycleBlock = ({ listOfKeeps }) => {
+  const [keepsList, setKeepsList] = useState([]);
+
+  useEffect(() => {
+    setKeepsList(listOfKeeps);
+  }, [listOfKeeps]);
```
Comment (melonholy) [inline google_keeps/src/components/RecycleComponent/RecycleBlock.jsx:23]: Why do you need extra local state?

File: google_keeps/src/components/pages/Keeps.jsx [hunk 2]
```diff
@@ -24,10 +28,19 @@ const Keeps = () => {
...
-      <Container>
-        <KeepCreate />
-        <KeepsList />
-      </Container>
+      <ContentWrapper>
+        <ContentSection>
+          <KeepCreate />
+          {!isKeepLoading &&
+          (!unGroupKeeps || !unGroupKeeps.length) &&
+          (!groupKeeps || !groupKeeps.length) ? (
```
Comment (melonholy) [inline google_keeps/src/components/pages/Keeps.jsx:36]: i suggest to move it in separate variable

**Review by melonholy — CHANGES_REQUESTED** [review]

File: google_keeps/src/components/pages/Keeps.jsx (hunk 2 above)
Comment (IgorOsiyuk) [reply google_keeps/src/components/pages/Keeps.jsx:32]: - [+]

File: google_keeps/src/components/RecycleComponent/RecycleBlock.jsx (hunk 1 above)
Comment (IgorOsiyuk) [reply google_keeps/src/components/RecycleComponent/RecycleBlock.jsx]: - [+]
removed unnecessary state and change it to checking props were got by components

**Review by melonholy — APPROVED** [review]

**Pull request merged**